test/extended/networking/network_segmentation.go

```diff
@@ -1508,8 +1508,26 @@ spec:
     topology: Layer3
     layer3:
       role: Primary
-      subnets: [{cidr: "10.100.0.0/16"}]
+      subnets: ` + generateCIDRforClusterUDN(oc)
```

Comment:
nit: I think we can use `generateCIDRforUDN` here, although, it uses different IPv6 CIDR, not sure if CUDN must use different CIDR.

Reply:
yea we could have, but I had to port what's upstream... let's leave this here for now, whoever cleans it up upstream can bring it downstream